app/lib/state_change_notifier.rb

```diff
@@ -1,85 +1,77 @@
 class StateChangeNotifier
-  def self.sign_up(candidate)
-    candidate_number = Candidate.where(hide_in_reporting: false).count
-
-    return unless (candidate_number % 25).zero?
-
-    candidate_number_is_significant = (candidate_number % 100).zero?
+  APPLICATION_OUTCOME = %i[rejected declined withdrawn recruited].freeze
 
-    human_readable_number = "#{ActiveSupport::NumberHelper.number_to_delimited(candidate_number)}#{candidate_number.ordinal}"
-
-    text = if candidate_number_is_significant
-             ":ultrafastparrot: The #{human_readable_number} candidate just signed up"
-           else
-             ":sparkles: The #{human_readable_number} candidate just signed up"
-           end
-
-    url = helpers.support_interface_candidate_url(candidate)
+  attr_reader :application_choice, :event
 
-    send(text, url)
+  def initialize(event, application_choice)
+    @event = event
+    @application_choice = application_choice
   end
 
-  def self.submit_application(application_form)
-    message = ":rocket: #{application_form.first_name}’s application has been sent to #{application_form.application_choices.map(&:provider).map(&:name).to_sentence}"
-    url = helpers.support_interface_application_form_url(application_form)
-
-    send(message, url)
-  end
+  def application_outcome
```

Comment:
Perhaps the method name should reflect the fact we're notifying something eg.
`notify_application_outcome`